doc/classes/AnimationNodeOneShot.xml

```diff
@@ -74,14 +74,14 @@
 		</member>
 		<member name="fadein_time" type="float" setter="set_fadein_time" getter="get_fadein_time" default="0.0">
 			The fade-in duration. For example, setting this to [code]1.0[/code] for a 5 second length animation will produce a cross-fade that starts at 0 second and ends at 1 second during the animation.
-			[b]Note:[/b] [AnimationNodeOneShot] transitions the current state after the end of the fading. When [AnimationNodeOutput] is considered as the most upstream, so the [member fadein_time] is scaled depending on the downstream delta. For example, if this value is set to [code]1.0[/code] and a [AnimationNodeTimeScale] with a value of [code]2.0[/code] is chained downstream, the actual processing time will be 0.5 second.
+			[b]Note:[/b] [AnimationNodeOneShot] transitions the current state after the end of the fading.
 		</member>
 		<member name="fadeout_curve" type="Curve" setter="set_fadeout_curve" getter="get_fadeout_curve">
 			Determines how cross-fading between animations is eased. If empty, the transition will be linear. Should be a unit [Curve].
 		</member>
 		<member name="fadeout_time" type="float" setter="set_fadeout_time" getter="get_fadeout_time" default="0.0">
 			The fade-out duration. For example, setting this to [code]1.0[/code] for a 5 second length animation will produce a cross-fade that starts at 4 second and ends at 5 second during the animation.
-			[b]Note:[/b] [AnimationNodeOneShot] transitions the current state after the end of the fading. When [AnimationNodeOutput] is considered as the most upstream, so the [member fadeout_time] is scaled depending on the downstream delta. For example, if this value is set to [code]1.0[/code] and an [AnimationNodeTimeScale] with a value of [code]2.0[/code] is chained downstream, the actual processing time will be 0.5 second.
```

Comment:
This deleted line is exactly what explains the change in behavior due to this PR compatibility breakdown. There is a migration path that multiplies the downstream TimeScale to the fading time of the NodeOneShot.

Reply:
FYI, the approach of PR #101788 is to adopt the delta of one of the two.

This PR respects both deltas by predicting the TimeScale (if the delta of the shot port animation is 0, the delta of the NodeOneShot side is preferred).

Reply:
Yes, that summary makes sense.

The part I didn't understand at first was that the time scale was calculated from the delta time of the downstream animation, and that a stopped animation node would have delta of 0 which has time scale of 0.

Reply:
If it is 0, the transition will not be performed because the delta to proceed the fade will be 0. Therefore, the fade must be proceeded with some delta.

And if a downstream time scale of greater than 0 is not taken into account, the problem of interrupted fades in https://github.com/godotengine/godot/issues/87009, may recur.

Reply:
"after the end of the fading." -> "after the fading has finished."